react-graphql-apollo-tutorial.md

```diff
@@ -527,7 +527,9 @@ const Profile = () =>
 export default Profile;
 {{< /highlight >}}
 
-The question is how to query data with GraphQL and Apollo Client now? The Apollo Client was provided in a previous section with React's Context API in a top level component. You have implicit access to it but never use it directly for standard queries and mutations. It says "standard" here, because there will be situation where you will use the Apollo Client instance directly while implementing this application. So how to query the data then? That's where React Apollo comes into play. The package gives you access to a Query component, which takes a query as prop and executes this query when it is rendered. That's the important part: It executes the query when it is rendered. Furthermore, it uses React's [render props](https://www.robinwieruch.de/react-render-props-pattern/) pattern. It uses a children as a function implementation where you can access the result of the query as argument eventually.
+Now we'll learn to query data with GraphQL and Apollo Client.  The Apollo Client was provided in a previous section with React's Context API in a top level component. You have implicit access to it, but never use it directly for standard queries and mutations. It says "standard" here, because there will be situations where you use the Apollo Client instance directly while implementing this application. 
+
+ The React Apollo package grants access to a Query component, which takes a query as prop and executes it when its rendered. That's the important part: it executes the query when props is rendered. It uses React's [render props](https://www.robinwieruch.de/react-render-props-pattern/) pattern, using a child as a function implementation where you can access the result of the query as an argument.
```

Comment:
"... it executes the query when props is rendered" to "... it executes the query when it is rendered"